core/include/usModuleSettings.h

```diff
@@ -43,11 +45,15 @@ US_BEGIN_NAMESPACE
  * - \e US_AUTOLOAD_PATHS A ':' (Unix) or ';' (Windows) separated list of paths
  *   from which modules should be auto-loaded.
  *
+ * \deprecated Use FrameworkFactory::newFramework(std::map<std::string, std::string> configuration) to configure the framework.
```

Comment:
Since this is deprecated now, we would need standardized framework properties for threading, logging, and storage configurations. These chould probably go into the `Framework` class and their values should start with `org.cppmicroservices`. Do not use `org.osgi` because of trademarks issues.